ops/ansible/roles/bfd-pipeline/defaults/main.yml

```diff
@@ -14,6 +14,9 @@ data_pipeline_ccw_rif_job_enabled: true
 rif_thread_multiple: 3
 rif_job_batch_size: 7
 rif_job_queue_size_multiple: 5
+rif_thread_multiple_claims: 25
+rif_job_batch_size_claims: 100
+rif_job_queue_size_multiple_claims: 10
```

Comment:
Looking better now. Where did these numbers come from by the way? Were these the results of testing and tuning, or are these just our first best guess and we'll tune later?

Reply:
These were recommended when the ticket was groomed. I think experience with pipeline loads in the past was the reason